output/xml/default.cppcode

```diff
@@ -304,6 +304,12 @@
 		<template name="event_handler_comment">
 			// Handlers for $basename events.
 		</template>
+		<template name="precompiled_header">
+			#ifnotnull $precompiled_header
+			@{
+				$precompiled_header
+			@}
+		</template>
```

Comment:
Regarding the position where that pch gets inserted i don't see a reason to use an extra template for that, can't it be just prepended to the `source_include` template?